<ui version="4.0" >
 <class>DialogNuevo</class>
 <widget class="QDialog" name="DialogNuevo" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferencias</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../recursos/recursos.qrc" >:/imagenes/preferencias.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>300</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title" >
    <string>Disposición</string>
   </property>
   <widget class="QRadioButton" name="radio_Aleatorio" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Aleatorio</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Manual" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>50</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QWidget" native="1" name="widget_4" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>110</width>
       <height>166</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Rojos</string>
        </property>
        <property name="buddy" >
         <cstring>nrojas</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="nrojas" >
        <property name="maximum" >
         <number>121</number>
        </property>
        <property name="value" >
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Azules</string>
        </property>
        <property name="buddy" >
         <cstring>nazules</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="nazules" >
        <property name="maximum" >
         <number>121</number>
        </property>
        <property name="value" >
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Amarillos</string>
        </property>
        <property name="buddy" >
         <cstring>namarillas</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="namarillas" >
        <property name="maximum" >
         <number>121</number>
        </property>
        <property name="value" >
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Rosas</string>
        </property>
        <property name="buddy" >
         <cstring>nrosas</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="nrosas" >
        <property name="maximum" >
         <number>121</number>
        </property>
        <property name="value" >
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Verdes</string>
        </property>
        <property name="buddy" >
         <cstring>nverdes</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSpinBox" name="nverdes" >
        <property name="maximum" >
         <number>121</number>
        </property>
        <property name="value" >
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>40</y>
     <width>163</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Número de colores</string>
      </property>
      <property name="buddy" >
       <cstring>spinBox_Colores</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_Colores" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="minimum" >
       <number>3</number>
      </property>
      <property name="maximum" >
       <number>5</number>
      </property>
      <property name="value" >
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>90</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Estilo</string>
   </property>
   <widget class="QRadioButton" name="radio_Continuo" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Continuo</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_Clasico" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Clásico</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../recursos/recursos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNuevo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNuevo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>326</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_Manual</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>90</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel" >
     <x>40</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
